.circleci/config.yml

```diff
@@ -99,11 +99,13 @@ jobs:
           name: Build image
           command: |
             docker build -t "$DOCKER_REPO":"${CIRCLE_TAG/v/}" .
+            docker build -t "$DOCKER_REPO":latest .
```

Comment:
This `build` can replace to `tag` maybe.
```
$ docker build -t lifullsetg/bucky-core:1.0 .
$ docker images
REPOSITORY               TAG     IMAGE ID        CREATED           SIZE
lifullsetg/bucky-core    1.0     a589dabd0fe2    11 seconds ago    604MB
$ docker tag lifullsetg/bucky-core:1.0 lifullsetg/bucky-core:latest
$ docker images
REPOSITORY               TAG       IMAGE ID        CREATED          SIZE
lifullsetg/bucky-core    1.0       a589dabd0fe2    2 minutes ago    604MB
lifullsetg/bucky-core    latest    a589dabd0fe2    2 minutes ago    604MB
```

Reply:
COOLLLL